docs/tutorials/catalog/simulated_image_demo.py

```diff
@@ -1,13 +1,19 @@
 """Simulates a galaxy of point sources and produces an image.
 """
-from astropy.units import Quantity
+import numpy as np
+import matplotlib.pyplot as plt
+
+import astropy.units as u
 from aplpy import FITSFigure
+
 from gammapy.astro import population
 from gammapy.datasets import FermiGalacticCenter
 from gammapy.image import make_empty_image, catalog_image
 from gammapy.irf import EnergyDependentTablePSF
 from gammapy.utils.random import sample_powerlaw
 
+np.random.seed(0)
```

Comment:
Please remove this call to `np.random.seed` and instead pass `random_state=0` to [make_base_catalog_galactic` and `sample_powerlaw` below.

In case you haven't seen it, we've now got this [FAQ entry on random numbers](https://gammapy.readthedocs.org/en/latest/development/index.html#random-numbers)